src/meds_torch/schemas/predict_schema.py

```diff
@@ -0,0 +1,148 @@
+# src/meds_torch/schemas/prediction_schema.py
+
+from collections import OrderedDict
+
+import polars as pl
+import pyarrow as pa
+
+# Constants for field grouping and ordering
+SCHEMA_FIELD_ORDER = OrderedDict(
```

Comment:
Is this not importable from meds-evaluation? If not, it should be, and please add an issue for it there.